yarn.lock

```diff
@@ -5421,7 +5421,7 @@ babel-plugin-jest-hoist@^24.9.0:
   dependencies:
     "@types/babel__traverse" "^7.0.6"
 
-babel-plugin-lodash@^3.3.4:
+babel-plugin-lodash@3.3.4, babel-plugin-lodash@^3.3.4:
```

Comment:
Something in our stack already depends on this? Hope it's not us :)

Reply:
We have a plugin that does this https://www.gatsbyjs.com/plugins/gatsby-plugin-lodash/ so I'll deprecate that later.